Test/WebTestCase.php

```diff
@@ -346,9 +353,51 @@ protected function postFixtureSetup()
     }
 
     /**
-     * Callback function to be executed after Schema restore.
+     * Callback function to be executed after Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function postFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function preFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed after save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
+     */
+    protected function postReferenceSave(ObjectManager $manager, AbstractExecutor $executor, $backupFilePath)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
      */
-    protected function postFixtureRestore()
```

Comment:
ok I am back.

yeah it would be good but I would like to avoid BC breaks unless we do a bigger refactoring. Now I am open to such a bigger refactoring (see https://github.com/liip/LiipFunctionalTestBundle/pull/136/files) in case someone has time to finish it up. Until then it would be better to revert.